doc/source/whatsnew/v0.22.0.txt

```diff
@@ -292,7 +292,7 @@ Plotting
 ^^^^^^^^
 
 - :func: `DataFrame.plot` now raises a ``ValueError`` when the ``x`` or ``y`` argument is improperly formed (:issue:`18671`)
--
+- Bug in :func: `DataFrame.plot` where the ``xtickvalues`` are overwritten (:issue:`#18687`)
 -
```

Comment:
no space after ``:func:``, can you add a bit more text on what a user would want to know here

issue is: ``(:issue:`18687`)`` (no ``#``)

Reply:
The line above mine has also a whitespace after `:func:`